### PR TITLE
Fixed boat not in garage after purchase

### PR DESCRIPTION
**Pull request by behindlife**
The correct property to stored all vehicles on garage ( This is correction to mysql ) the script can stored boats property.

### CODE REVIEW

**Comment by KoolAiddTV** [comment]
hey @behindlife when I try to use this fix I'm getting a error in my console.
`You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'stored)`
the line that's causing it is 
`MySQL.Async.execute('INSERT INTO owned_vehicles (owner, plate, vehicle, type, stored) VALUES (@owner, @plate, @vehicle, @type, @stored)', {`

any idea on to fix this?

**Comment by MonsterZockerHD** [comment]
This fixes the bug with you can't park out a vehicle!

File: server/main.lua [hunk 1]
```diff
@@ -28,11 +28,12 @@ ESX.RegisterServerCallback('esx_boat:buyBoat', function(source, cb, vehicleProps
 		if xPlayer.getMoney() >= price then
 			xPlayer.removeMoney(price)
 
-			MySQL.Async.execute('INSERT INTO owned_vehicles (owner, plate, vehicle, type) VALUES (@owner, @plate, @vehicle, @type)', {
+			MySQL.Async.execute('INSERT INTO owned_vehicles (owner, plate, vehicle, type, stored) VALUES (@owner, @plate, @vehicle, @type, @stored)', {
```
Comment (ElPumpo) [inline server/main.lua]: Surround stored with ` character since it is a reserved word

File: server/main.lua [hunk 2]
```diff
@@ -28,11 +28,12 @@ ESX.RegisterServerCallback('esx_boat:buyBoat', function(source, cb, vehicleProps
...
 				['@owner']   = xPlayer.identifier,
 				['@plate']   = vehicleProps.plate,
 				['@vehicle'] = json.encode(vehicleProps),
-				['@type']    = 'boat'
+				['@type']    = 'boat',
+				['@stored']  = 1
```
Comment (ElPumpo) [inline server/main.lua]: Instead of `1` use `true`

**Pull request closed without merging**